lib/nfc/protocols/mf_classic/mf_classic_poller_sync_api.c

```diff
@@ -0,0 +1,524 @@
+    MfClassicPoller* poller,
+    MfClassicPollerContextData* data) {
+    MfClassicError error = MfClassicErrorNone;
+
+    do {
+        error = mf_classic_async_auth(
+            poller,
+            data->read_value_context.block_num,
+            &data->read_value_context.key,
+            data->read_value_context.key_type,
+            NULL);
+        if(error != MfClassicErrorNone) break;
+
+        MfClassicBlock block = {};
+        error = mf_classic_async_read_block(poller, data->read_value_context.block_num, &block);
+        if(error != MfClassicErrorNone) break;
+
+        if(!mf_classic_block_to_value(&block, &data->read_value_context.value, NULL)) {
+            error = MfClassicErrorProtocol;
+            break;
+        }
+
+        error = mf_classic_async_halt(poller);
+        if(error != MfClassicErrorNone) break;
+
+    } while(false);
+
+    return error;
+}
+
+static MfClassicError mf_classic_poller_change_value_handler(
+    MfClassicPoller* poller,
+    MfClassicPollerContextData* data) {
+    MfClassicError error = MfClassicErrorNone;
+
+    do {
+        error = mf_classic_async_auth(
+            poller,
+            data->change_value_context.block_num,
+            &data->change_value_context.key,
+            data->change_value_context.key_type,
+            NULL);
+        if(error != MfClassicErrorNone) break;
+
+        error = mf_classic_async_value_cmd(
+            poller,
+            data->change_value_context.block_num,
+            data->change_value_context.value_cmd,
+            data->change_value_context.data);
+        if(error != MfClassicErrorNone) break;
+
+        error = mf_classic_async_value_transfer(poller, data->change_value_context.block_num);
+        if(error != MfClassicErrorNone) break;
+
+        MfClassicBlock block = {};
+        error = mf_classic_async_read_block(poller, data->change_value_context.block_num, &block);
+        if(error != MfClassicErrorNone) break;
+
+        error = mf_classic_async_halt(poller);
+        if(error != MfClassicErrorNone) break;
+
+        if(!mf_classic_block_to_value(&block, &data->change_value_context.new_value, NULL)) {
+            error = MfClassicErrorProtocol;
+            break;
+        }
+
+    } while(false);
+
+    return error;
+}
+
+static const MfClassicPollerCmdHandler mf_classic_poller_cmd_handlers[MfClassicPollerCmdTypeNum] = {
+    [MfClassicPollerCmdTypeCollectNt] = mf_classic_poller_collect_nt_handler,
+    [MfClassicPollerCmdTypeAuth] = mf_classic_poller_auth_handler,
+    [MfClassicPollerCmdTypeReadBlock] = mf_classic_poller_read_block_handler,
+    [MfClassicPollerCmdTypeWriteBlock] = mf_classic_poller_write_block_handler,
+    [MfClassicPollerCmdTypeReadValue] = mf_classic_poller_read_value_handler,
+    [MfClassicPollerCmdTypeChangeValue] = mf_classic_poller_change_value_handler,
+};
+
+static NfcCommand mf_ultralgiht_poller_cmd_callback(NfcGenericEvent event, void* context) {
```

Comment:
Should be `mf_classic_poller_cmd_callback`? Also, typo `ultralgiht` in same function in `lib/nfc/protocols/mf_ultralight/mf_ultralight_poller_sync_api.c`

Reply:
Also, in continuation of typos: `MF_ULTRALIGTH_CMD_READ_SIG` (should be `MF_ULTRALIGHT_CMD_READ_SIG`)

Reply:
Continuing, `mf_ultraligt_format_mirror_data` should be `mf_ultralight_format_mirror_data`. `mf_ultraligt_mirror_prepare_emulation` should be `mf_ultralight_mirror_prepare_emulation`

Reply:
`mf_classic_listnener_auth_first_part_handler` -> `mf_classic_listener_auth_first_part_handler`
`MF_ULTRALIGHT_POLLER_STANDART_FWT_FC` -> `MF_ULTRALIGHT_POLLER_STANDARD_FWT_FC`

Reply:
Thanks! Will be fixed soon